packages/js/src/settings/helpers/submit.js

```diff
@@ -1,8 +1,7 @@
 import { dispatch, select } from "@wordpress/data";
 import { __ } from "@wordpress/i18n";
-import { forEach, get, includes, isArray, isNumber, isObject, omit } from "lodash";
+import { forEach, get, includes, isArray, isObject, omit } from "lodash";
 import { STORE_NAME } from "../constants";
-import { submitUserSocialProfiles } from "./user-social-profiles";
```

Comment:
In this file on line 22 there is also a reference to `person_social_profiles` should that also be removed?

Reply:
Good call!